### ISSUE TITLE
Insufficient contrast on link text

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15276]
Insufficient contrast on link text

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Block Types
    -   Media Dialog

#### Issue description

When the "Featured Image" modal is open, there are two controls coded
as links which have insufficient color contrast, below the minimum
threshold of 4.5:1 for text:

-   "Upload Files" and "Media Library" blue hover color: blue (#00a0d2) on white (#fff): **3.01:1**

Sufficient color contrast is important for users who have low-vision or
are color-blind, because text with a low contrast ratio may be difficult
or impossible for such users to see.

##### Issue Code

```
    .media-frame a:active, .media-frame a:hover {
        color: #00a0d2;
    }

```

#### Remediation Guidance

Darken the hover color to match the focus color, or otherwise to provide
a contrast ratio of 4.5:1 or greater.

##### Recommended Code

```
    .media-frame a:active, .media-frame a:hover {
        color: #124964;
    }
```

#### Relevant standards

-   [1.4.3 Contrast (Minimum) (Level
    AA)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-43 in Tenon's report

**Comment by afercia** [comment]
Worth noting this applies to _all_ the links in WordPress but _only_ to the blue used for the _hover_ state.

I'd like to discuss this a bit more in depth, as I'm not 100% sure the hover state needs sufficient contrast,

**Comment by melchoyce** [comment]
Full report screenshot: 
![image](https://user-images.githubusercontent.com/2846578/57186434-164ef800-6e94-11e9-8cdc-f15e98a631d4.png)

**Comment by StommePoes** [comment]
I would say hover needs sufficient contrast for the cases of low-viz screen-mag users for whole avoiding a hovered state can be difficult (depending on zoom level and the size of the control of course).

**Comment by afercia** [comment]
Closing, as moved to Trac https://core.trac.wordpress.org/ticket/47157
